imports/documentGeneration.js

```diff
@@ -0,0 +1,119 @@
+            }
+            topic.labels = aTopicObj.getLabelsString(topic);
+        });
+    }
+
+    static getAttachmentsFromMinute(minuteID) {
+        if (minuteID) {
+            let attachments = Attachment.findForMinutes(minuteID).fetch();
+            attachments.forEach((file) => {
+                let usr = Meteor.users.findOne(file.userId);
+                return file.username = usr.username;
+            });
+            return attachments;
+        }
+    }
+
+    static getDocumentData(context) {
+        let presentParticipants = context._participants.filter(participant => {
+            return participant.present;
+        });
+
+        let absentParticipants = context._participants.filter(participant => {
+            return !participant.present;
+        });
+
+        let discussedTopics = context._topics.filter(topic => {
+            return !topic.isOpen;
+        });
+
+        let outstandingTopics = context._topics.filter(topic => {
+            return (topic.isOpen && !topic.isSkipped);
+        });
+
+        return {
+            minutesDate: context._minute.date,
+            minutesGlobalNote: context._minute.globalNote,
+            meetingSeriesName: context._meetingSeries.name,
+            meetingSeriesProject: context._meetingSeries.project,
+            meetingSeriesURL: GlobalSettings.getRootUrl('meetingseries/' + context._meetingSeries._id),
+            minuteUrl: GlobalSettings.getRootUrl('minutesedit/' + context._minute._id),
+            presentParticipants: context._userArrayToString(presentParticipants),
+            absentParticipants: context._userArrayToString(absentParticipants),
+            informedUsers: context._userArrayToString(context._informed),
+            participantsAdditional: context._minute.participantsAdditional,
+            discussedTopics: discussedTopics,
+            skippedTopics: outstandingTopics,
+            finalizedVersion: context._minute.finalizedVersion,
+            attachments: this.getAttachmentsFromMinute(context._minute._id)
+        };
+    }
+
+    static addHelperForHTMLMail(template, context) {
+        template.addHelper('hasLabels', function() {
+            return (this.labels.length > 0);
+        });
+        template.addHelper('formatLabels', function(parentTopicId) {
+            let parentTopic = new Topic(context._minute, parentTopicId);
+            let infoItemId = this._id;
+            let infoItem = InfoItemFactory.createInfoItem(parentTopic, infoItemId);
+            let labels = infoItem.getLabels(context._minute.parentMeetingSeriesID());
+            let result = '';
+            let first = true;
+            labels.forEach(label => {
+                if (first) {
+                    first = false;
+                } else {
+                    result += ', ';
+                }
+                result += '#'+label.getName();
+            });
+            return result;
```

Comment:
Please consider replacing the last 10 lines with

    const result = labels.map(label => '#' + label.getName()).join(', ')

This way you get rid of the state (`first` & `result`) and repeated string concatenation (`+=`).